### DISCUSSION TITLE
method appears to be non-existent

### DISCUSSION THREAD

**Discussion by eduardonwa** [filamentphp/filament#7886] [Help]
hi all, i just installed the package, and updated my Post model accordingly, everything went okay, migrations, etc but im getting this message:
```Method Filament\Forms\Components\SpatieMediaLibraryFileUpload::getUploadedFileUsing does not exist.``` so i tried to update composer to no problem. 
im using version : * 10.11.3 of the plugin and im currently at v2.17.52 of filament 

my form is looking like this
```
  SpatieMediaLibraryFileUpload::make('thumbnail')
      ->collection('thumbnails')
      ->multiple()
      ->responsiveImages(),
```
table,
```
  SpatieMediaLibraryImageColumn::make('thumbnail'),
```

app_url is the same on the browser and .env file

https://flareapp.io/share/VmeMZ8qm

the below question shows the same error, unless im not being aware of something i don't know what else to try. I've ran the require command again and then tried updating composer to no problem but the error persists, what is wrong? Thanks. 

https://github.com/filamentphp/filament/issues/5864

**Comment by pboivin** [comment]
I'm not sure but try to pin the package to the 2.x version specifically:
```
composer require filament/spatie-laravel-media-library-plugin:"^2.0"
```

**Reply by eduardonwa** [reply]
Great! Now i cant seem to display them on the table or elsewhere, using ```$post->media```
Is there something wrong here:

``` 
    public function registerMediaConversions(Media $media = null): void
    {
        $this
            ->addMediaConversion('preview')
            ->fit(Manipulations::FIT_CROP, 300, 300)
            ->nonQueued();
    }
```

**Comment by eduardonwa** [comment]
i was able to display an image on my views (still can't manage to display them on the table) but it's not a responsive image. how can i make it fetch for a responsive item? 

```src="{{ $latestPost->getFirstMediaUrl('thumbnails') }}"``` 

going by the size of it (326kb) on a mobile viewport, shouldn't it display a smaller image in terms of size? thanks,
